hack/helm.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
```

Comment:
I was expecting a mark target at the top level `make chart` or `make helm` something like that.